pages/favorites/index.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+import SpotCard from "../../components/SpotCard.js";
+
+export default function FavoriteSpots({ spots, toggleBookmark }) {
+  return (
+    <StyledBookmarkSection>
+      <ul>
+        {spots?.map((spot) => {
+          if (spot.isBookmark === true) {
+            return (
+              <SpotCard
+                spot={spot}
+                key={spot.name}
```

Comment:
please use `spot.id` as key instead of the name, because the name might not be unique.

```suggestion
                key={spot.id}
```